src/components/existing-request/existing-request-display.vue

```diff
@@ -44,8 +44,29 @@
                 <v-col cols="12" class="request-status">
                   <span>Request Status:</span>
                   &nbsp;
-                  <span :class="isNotPaid ? 'app-red' : isPaymentProcessing ? 'app-green' : ''">
-                    {{ requestStatusText }}</span>
+                  <span
+                    :class="isRefundRequested ? '' :
+                            isNotPaid ? 'app-red' :
+                            isPaymentProcessing ? 'app-green' : ''">
```

Comment:
This chained ternary is a little hard to follow. 
Have you looked into class binding the colors conditionally?

Reply:
yes, actually I changed isPaymentProcessing logic and revert this part back the original. Wil come in the next commit.

Reply:
Sounds good!